src/main/scala/loamstream/cli/ExecutionInfo.scala

```diff
@@ -43,8 +43,7 @@ object ExecutionInfo {
     
     val result = execution.result
     
-    val stdout = execution.outputStreams.map(_.stdout).getOrElse("<Not produced>")
-    val stderr = execution.outputStreams.map(_.stderr).getOrElse("<Not produced>")
+    val jobDir = execution.jobDir.map(_.toString).getOrElse("<None>")
```

Comment:
Not sure what this string is being used for, but since there was CSV code above, if this is intended to be written to the CSV, I'd recommend changing it to `"NA"` or an empty string. If it's just for debug output, then whatever. 😄

Reply:
It's the output of some CLI flag that I don't know if anyone uses. :\